modules/openapi-generator/src/main/resources/JavaSpring/pojo.mustache

```diff
@@ -140,6 +139,9 @@ public class {{classname}}{{#parent}} extends {{{parent}}}{{/parent}}{{^parent}}
   {{/maximum}}
    * @return {{name}}
   */
+  {{#jackson}}
+  @JsonProperty("{{baseName}}")
```

Comment:
Can we move the `@JsonProperty` directly above the getter method? This would avoid empty lines in the generated model classes.